tedana/workflows/tedana.py

```diff
@@ -164,19 +165,46 @@ def _get_parser():
         ),
         default="kundu",
     )
+    optional.add_argument(#####BTBTBT
+        "--ica_method",
+        dest="ica_method",
+        help=(
+            "The applied ICA method. If set to fastica the FastICA "
+            "from sklearn library will be run once with the seed value. "
+            "robustica will run FastICA n_robust_runs times and and uses "
+            "clustering methods to overcome the randomness of the FastICA "
+            "algorithm. If set to robustica the seed value will be ignored."
+            "If set to fastica n_robust_runs will not be effective."
+        ),
+        choices=["robustica", "fastica"],
+        default="robustica",
+    )
     optional.add_argument(
         "--seed",
         dest="fixed_seed",
         metavar="INT",
         type=int,
-        help=(
+        help=( ##BTBTBT
             "Value used for random initialization of ICA "
-            "algorithm. Set to an integer value for "
-            "reproducible ICA results. Set to -1 for "
+            "algorithm when ica_mthods is set to fastica. Set to an integer value for "
+            "reproducible ICA results with fastica. Set to -1 for "
             "varying results across ICA calls. "
         ),
         default=42,
     )
+    optional.add_argument(#####BTBTBT
+        "--n_robust_runs",
+        dest="n_robust_runs",
+        type=int,
+        help=(
+            "The number of times robustica will run."
+            "This is only effective when ica_mthods is "
+            "set to robustica."
+
+        ),
+        ##choices=range(2,100),
```

Comment:
Delete

Reply:
The space has been removed.

Reply:
This comment was targeting the commented line of code. While this happens all the time when evaluating possible code, such should not appear once you get to the point of proposing upstream changes.